configure.ac

```diff
@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
 AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_SUBST(RPM_VERSION, AC_PACKAGE_VERSION)
```

Comment:
Why not just leave this out and use VERSION which is standard and we get substituted for free?